modules/EC2/main.tf

```diff
@@ -0,0 +1,68 @@
+locals {
+  owner         = "${var.identifier} - nclouds"
+  allowed_ports = [22, 443, 80, 8080]
+}
+
+data "aws_ami" "myami_ss" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["myami-*"]  # Replace with the appropriate AMI name
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  filter {
+    name   = "architecture"
+    values = ["x86_64"]
+  }
+
+}
+
```

Comment:
Great! I see the change here
nice work @shivani121993

Reply:
thankyou Abel